Library/Homebrew/cmd/update.rb

```diff
@@ -131,6 +133,10 @@ def update
     Descriptions.update_cache(report)
   end
 
+  def shortenRevision(revision)
```

Comment:
This needs to be named `shorten_revision` to match our (and the usual Ruby) naming convention for methods. It should also go below the `private`, I think.